fmn/database/model/destination.py

```diff
@@ -29,20 +34,34 @@ class Destination(Base):
     protocol = Column(String(length=255), nullable=False)
     address = Column(UnicodeText, nullable=False)
 
-    def generate(self, message: "Message") -> "Notification.content":
+    async def generate(self, message: "Message") -> "Notification.content":
         app_name = f"[{message.app_name}] " if message.app_name else ""
         url = message.url if message.url else ""
         if self.protocol == "email":
+            extra = await get_extra(message)
             return {
                 "headers": {
                     "To": self.address,
                     "Subject": f"{app_name}{message.summary}",
                 },
-                "body": f"{str(message)}\n{url}",
+                "body": f"{str(message)}\n{url}\n{extra}",
             }
         elif self.protocol == "irc":
             return {"to": self.address, "message": f"{app_name}{message.summary} {url}"}
         elif self.protocol == "matrix":
             return {"to": self.address, "message": f"{app_name}{message.summary} {url}"}
         else:
             raise ValueError(f"Unknown destination protocol: {self.protocol}")
+
+
+async def get_extra(message: "Message"):
+    http_client = AsyncClient(timeout=10)
+    if getattr(message, "patch_url", None) is not None:
```

Comment:
```suggestion
    if hasattr(message, "patch_url"):
```

Reply:
Well, not only, I also want to abort if `message.patch_url is None`

Reply:
And if `message.patch_url` is an empty string? Then it could be `if getattr(message, "patch_url):`.

Reply:
Ah, understood.